### ISSUE TITLE
could you please add the proquest lookup engine?

### ISSUE THREAD

**Issue by fredericky123** [egh/zotero-lookup-engines#24]
Heis is the website:
https://search.proquest.com/

Thanks!

**Comment by egh** [comment]
I can add this if you provide a template. I'm not sure what it should look like. Thanks!

**Comment by fredericky123** [comment]
I am sorry I dont have a template.
When I try to search a paper, like "Financing Innovation: Evidence from R&D Grants" in that database, this is the link:
https://search.proquest.com/results/E594DB4B4801456DPQ/1?accountid=14426

However, I  cannot interpret the link.

**Comment by egh** [comment]
Yes, unfortunately I can't figure that out at all :) I'll see what I can do

**Comment by fredericky123** [comment]
Thanks all the same!
